core-tests/shared/src/test/scala/zio/ClockSpec.scala

```diff
@@ -37,6 +39,45 @@ object ClockSpec extends ZIOBaseSpec {
       assertZIO(ZIO.collectAll(Chunk(zioEuropeAmsterdamClock.localDateTime, zioAmericaPhoenixClock.localDateTime)))(
         isDistinct
       )
-    }
+    },
+    suite("TestClock")(
+      test("should accept any Instant") {
+        check(Gen.instant) { instant =>
+          for {
+            _   <- TestClock.setInstant(instant)
+            now <- Clock.instant
+          } yield assertTrue(now == instant)
+        }
+      },
+      test("should allow sleep") {
```

Comment:
Do we need this test and the one below it now?

Reply:
Not really. I just created them to narrow down some issue with `Scheduler` tests.
Deleted.